### ISSUE TITLE
Add Validation Data

### ISSUE THREAD

**Issue by djwalker200** [djwalker200/TreeNN#3]
Split the MNIST training dataset to include a validation set.
